unlock-app/src/propTypes.js

```diff
@@ -30,6 +30,8 @@ export const conversion = PropTypes.objectOf(PropTypes.number)
 
 export const children = PropTypes.shape({})
 
+export const callback = PropTypes.func
```

Comment:
Is that necessary? I mean, a `callback` is nothing specific to Unlock?

Reply:
I wasn't sure, since there are some other logical aliases in the file :) I'm happy using .func